cypress/e2e/user-flows/scenario_008.cy.js

```diff
@@ -0,0 +1,109 @@
+import { SimulatorPage } from "../../pages/simulator/simulatorPage";
```

Comment:
I think we're trying not to name our files that way.
Maybe `scenario_08_edit_mock_from_simulator` or something like that owuld be cool.

Have some kind of convention. (this could also be a topic for E2E tech debt 😄 )